xaynetwork/xayn_discovery_app
# TB-4022 TB-4067 Integrate experimentation engine

### What 🕵️ 🔍

- What does the pull request solve?
Integrate experimentation engine with xayn app.

- What exactly was added or modified?
Alter feature map depending on features from fetched experiments.

----------

### How to test (please adjust template) 🥼 🔬
- Check onboarding sheets are showing when the user is subscribed to the onboarding experiment (you can check from mixpanel).
- Check that onboarding sheets are not showing when the user is not subscribed to the onboarding experiment (you can check from mixpanel).

----------

### References 📝 🔗

- [JIRA](https://xainag.atlassian.net/browse/TB-4022)
- [JIRA](https://xainag.atlassian.net/browse/TB-4067)

----------
